Vizzuality/i2i
# Feature/layers icons

Description of the PR

New styles in sidebar:
  - Layer items menu adapted to new design.
  - Analysis icon added to sector layers.

> This PR is against feature/sidebar-icons branch because that's where icomoon was added

## Testing instructions

Go to data portal/geospatial data/ click on any of the countries, check layers tab (different types)

E.g. http://localhost:3000/data-portal/BGD/fsp-maps?lat=23.56398712845123&layers=%5B%7B%22id%22%3A%22Mobile%20Money%20Agent%22%2C%22opacity%22%3A1%2C%22visibility%22%3Atrue%2C%22visualizationType%22%3A%22normal%22%7D%5D&lng=90.17578125000001&zoom=3

## Pivotal Tracker

https://www.pivotaltracker.com/story/show/169184419
